doc/prepare_raspberrypi.rst

```diff
@@ -53,6 +70,66 @@ Log in to it::
 
 Enter the password mentioned just above.
 
+Installation using Windows
+--------------------------
+This section describes how to install the Raspberry Pi OS
+and how to enable ssh and serial console so that the Raspberry Pi can be 
+used in headless mode without a display and keyboard connected.
+
+Step 1. Write Raspberry Pi OS image to SD card using Rasberry Pi Imager.
+
+* Download and install Raspberry Pi Imager from 
+  https://www.raspberrypi.org/software/
+* Start Raspberry Pi Imager
+* In the Select OS dialog choose full version
+* Select SD-card
+* Press Write
+
+Step 2. Initial configuration of Raspberry Pi OS.
+
+* Eject SD-card 
+* Reinsert SD-card in windows PC. The SD-card will be shown as external drive named "boot".
+* Enable ssh by creating an empty file named ''ssh'' in the root folder of "boot". 
+  The windows file explorer can be used for this.
+  Note that the file "ssh" shall not have a txt file extension.
+* Enable serial port console. 
+  Open ``config.txt`` in root folder of ''boot'' using Notepad.
+  Add the line ''enable_uart=1'' to the end of the file.
+  Save file and close Notepad.
+* Eject SD-card
+
+Step 3. Start Rasberry Pi
+
+* Insert SD-card and power on Raspberry Pi.
+* Login (preferably using serial console) with default user ''pi'' and password ''raspberry''.
+
+Step 4. Network configuration.
+
+Use the nano editor to edit the configuration files as described below. 
+For example to edit the ''/etc/dhcpcd.conf''::
+
+    sudo nano /etc/dhcpcd.conf
+
```

Comment:
`Save the file in nano by pressing ``CTRL-X``, then ``Y`` and ``Enter``.`

Reply:
Done